### ISSUE TITLE
Alerts for voltage needle and/or lightbulb

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/faradays-law#207]
For https://github.com/phetsims/QA/issues/581. Discovered while testing iPadOS VO, but applies more generally to screen reader script and recorded with a11y view.

As I was testing the screen reader dialog, I found it disconcerting that nothing was read while moving the magnet. The information could be somewhat gleaned with the sound effect of the lightbulb. But information about the needle direction, brightness of the lightbulb, where the magnet was passing through, or any information if the sim was muted, was lost. I think some alerts should be added to provide this information.

![noalerts](https://user-images.githubusercontent.com/41024075/101554514-06eb3700-3974-11eb-9d79-db1d12397c96.gif)

**Comment by emily-phet** [comment]
Thanks @KatieWoe! We agree, full description for this sim would be great. We didn't have the personnel resources to complete this sim's description this year, and so removed the incomplete parts. This sim, once published in this round, will not be listed as having the interactive description feature, so any description you hear (e.g., navigating the state descriptions such as object names and headings) is kind of "bonus" content for now.
